<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AirportStatistic</class>
 <widget class="QWidget" name="AirportStatistic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Airport statistic</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QLabel" name="lb_Airport">
     <property name="text">
      <string>b_Airport</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_Year">
      <attribute name="title">
       <string>Year</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QWidget" name="wid_YearChart" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Month">
      <attribute name="title">
       <string>Month</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QComboBox" name="cb_Months">
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <property name="maxVisibleItems">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="wid_MonthChart" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pb_Close">
     <property name="text">
      <string>pb_Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
